### DISCUSSION TITLE
SIG-Ops Meeting Poll

### DISCUSSION THREAD

**Discussion by JustFinchy** [o3de/sig-operations#36] [General]
Hello, 

Since we have had very low numbers of people attending the meetings, I wanted to redo the time sections in hopes to fit with everyone calendar better. 

Please drop your timezone and times available as well as preferred days. Once I have a few, I will compile, build a voting setup and then we can move from there. I will put mine as an example below. 

Hours available 
9AM to 8PM PST

Days Available 
Sunday to Saturday

**Comment by sptramer** [comment]
* Monday: 11AM-12PM PT
* Tuesday: 9AM-10AM PT
* Thursday: 9AM-10AM, 2-3PM, 4-5PM (PT)
* Friday: 2PM-4PM PT

In the past we've been using [Doodle](https://doodle.com/en/) for time polling.
